File: OpenAPIGenerationSample/WeatherForecast.cs
namespace OpenAPIGenerationSample
{
    /// <summary>
    /// Data about a Weather Forecast
    /// </summary>
    public class WeatherForecast
    {
        public DateOnly Date { get; set; }

        public int TemperatureC { get; set; }

        public int TemperatureF => 32 + (int)(TemperatureC / 0.5556);

        /// <summary>
        /// A one-word description of the weather
        /// </summary>
        public string? Summary { get; set; }

        public WeatherType WeatherType { get; set; }
    }
}

File: OpenAPIGenerationSampleApp/src/app/_api/models/weather-forecast.ts
/* tslint:disable */
/* eslint-disable */
import { DateOnly } from '../models/date-only';
import { WeatherType } from '../models/weather-type';

/**
 * Data about a Weather Forecast
 */
export interface WeatherForecast {
  date?: DateOnly;

  /**
   * A one-word description of the weather
   */
  summary?: string | null;
  temperatureC?: number;
  temperatureF?: number;
  weatherType?: WeatherType;
}
